web/src/components/ProjectsPanel.vue

```diff
@@ -9,109 +9,327 @@
-      }
+        if (newValue && newValue.length > 0) {
+          this.selectedVersion = newValue[0];
+          this.getInfoPackage();
+        }
+      },
     },
     projects(newValue) {
       let oldSelectedProjects = this.selectedProjects;
       this.selectedProjects = [];
-      oldSelectedProjects.forEach(x => {
+      oldSelectedProjects.forEach((x) => {
         let previouslySelectedProject = newValue.find(
-          p => p.projectName == x.projectName
+          (p) => p.projectName == x.projectName
         );
         if (previouslySelectedProject) {
           this.selectedProjects.push(previouslySelectedProject);
         }
       });
-    }
+    },
   },
   props: {
     projects: Array,
     versions: Array,
     packageId: String,
-    packageAuthors: Array
+    packageAuthors: Array,
   },
   methods: {
     install(projectsToInstall) {
       if (projectsToInstall && projectsToInstall.length > 0) {
         this.$emit("install", {
           selectedProjects: projectsToInstall,
-          selectedVersion: this.selectedVersion
+          selectedVersion: this.selectedVersion,
         });
       }
     },
-    uninstall(projectsToUninstall) {      
+    uninstall(projectsToUninstall) {
       if (projectsToUninstall && projectsToUninstall.length > 0) {
         this.$emit("uninstall", {
-          selectedProjects: projectsToUninstall
+          selectedProjects: projectsToUninstall,
         });
       }
     },
     canInstall() {
       let safeSelectedVersion = this.selectedVersion;
-      return (this.selectedProjects 
-      && this.selectedProjects.length > 0
-      && this.selectedProjects.every(function (p) {
-        return (p.version === null 
-        || (safeSelectedVersion !== null && p.version !== safeSelectedVersion));        
-      }));
+      return (
+        this.selectedProjects &&
+        this.selectedProjects.length > 0 &&
+        this.selectedProjects.every(function (p) {
+          return (
+            p.version === null ||
+            (safeSelectedVersion !== null && p.version !== safeSelectedVersion)
+          );
+        })
+      );
     },
-    canUninstall() {      
+    canUninstall() {
       let safeSelectedVersion = this.selectedVersion;
-      return (this.selectedProjects 
-      && this.selectedProjects.length > 0
-      && this.selectedProjects.every(function (project) {        
-        return (project.version && project.version === safeSelectedVersion)        
-      }));
+      return (
+        this.selectedProjects &&
+        this.selectedProjects.length > 0 &&
+        this.selectedProjects.every(function (project) {
+          return project.version && project.version === safeSelectedVersion;
+        })
+      );
+    },
+    getInfoPackage() {
+      this.packageMetadata = null;
+      const url = `https://api.nuget.org/v3/registration5-gz-semver2/${this.packageId}/index.json`.toLowerCase();
```

Comment:
it can't be a fixed url, as the application uses many sources. Also authorization is required